content/en/agent/faq/agent-v6-changes.md

```diff
@@ -0,0 +1,564 @@
+  text: "Upgrade to Datadog Agent v6"
+- link: "agent/faq/how-datadog-agent-determines-the-hostname/"
+  tag: "Documentation"
+  text: "How does Datadog determine the Agent hostname?"
+---
+
+## Overview
+
+Datadog Agent v6 contains many changes compared to previous Agent versions. The changes and deprecations are detailed in the sections below.
+
+## Features
+
+The following Agent v5 features are **not available** in Agent v6:
+
+* [Using the Agent as a Proxy][1]
+* [Custom emitters][2]
+* [Dogstream][3]
+* [go-metro][4]
+* Graphite support
+
+## Configuration
+
+Prior versions of the Agent stored configuration files in `/etc/dd-agent`. For Agent v6.0+, configuration files are stored in `/etc/datadog-agent`.
+
+{{< tabs >}}
+{{% tab "Agent" %}}
+
+The [Agent's main configuration file][1] has transitioned from **INI** to **YAML** format to support complex configurations and provide a consistent experience across the Agent and checks.
+
+Agent v5 `datadog.conf` --> Agent v6 `datadog.yaml`
+
+To transition between Agent configuration paths and formats, use the Agent command:
+```bash
+sudo -u dd-agent -- datadog-agent import
+```
+
+This command parses an existing `datadog.conf` and converts supported parameters to the new format in `datadog.yaml`. The command also copies configuration files for checks that are currently enabled. For more details, see [Upgrade to Datadog Agent v6][2].
+
+#### Configuration options
+
+The following Agent configuration options were changed or removed in Agent v6. Configuration options removed were either superseded by other options, or related to features that work differently from previous versions.
+
+##### Changed
+
+| Previous Name               | Updated Name                 | Notes                                                                                             |
+|-----------------------------|------------------------------|---------------------------------------------------------------------------------------------------|
+| `proxy_host`                | `proxy`                      | Proxy settings are now expressed as a list of URIs. See the [proxy][3] documentation for details. |
+| `collect_instance_metadata` | `enable_metadata_collection` | Enables metadata collection                                                                       |
+| `collector_log_file`        | `log_file`                   |                                                                                                   |
+| `syslog_host`               | `syslog_uri`                 | The Syslog configuration is now expressed as a URI.                                               |
+|                             | `syslog_pem`                 | Syslog configuration client certificate for TLS client validation.                                |
+|                             | `syslog_key`                 | Syslog configuration client private key for TLS client validation.                                |
```

Comment:
There is no previous name for those two parameters ?

Reply:
Not that was available in the current docs